### DISCUSSION TITLE
Example usage of Filters.where(javascriptExpr)?

### DISCUSSION THREAD

**Discussion by eheimburg** [MorphiaOrg/morphia#2328] [General]
**Please ask your question and provide whatever background or supporting information you think might be relevant:**

I'm unable to find a way to use `Filters.where(expr)` to execute a javascript query. No matter how I try to use it, I get this exception:

`dev.morphia.query.ValidationException: Could not resolve path '' against 'gorgon.entities.Player'.  Unknown path element: ''.`

I'm not sure what path it's talking about or why it's an empty string. I'm not sure if there's a bug, or if I'm misunderstanding how to use it.

** Please complete the following information: **
    Server Version:
    Driver Version:
    Morphia Version: 2.3

**Comment by evanchooly** [comment]
well, the good news is that sound like a bug to me.  By default all those filter expressions get validated but they shouldn't be for this particular one.  The fix for that shouldn't be terrible.  However, I'm getting on a plane tomorrow morning for a conference but I'll try to get that fixed soon.

The bad news is that I have little practical experience using that particular query feature.  I was pretty sure it had been deprecated on the server but turns out even that is a bit more complicated (it has to do with old BSON types being deprecated rather than the entire feature).  All that to say, I can try to help work out any wrinkles once I get the validation fixed but I can't make many promises there.
